src/pages/CustomerInvoicePage.js

```diff
@@ -201,23 +257,54 @@ export class CustomerInvoicePage extends GenericPage {
   };
 
   renderCell = (key, transactionItem) => {
+    const { transaction } = this.props;
+    const { totalQuantity, totalDoses } = transactionItem;
+    const { isFinalised } = transaction;
+    const emptyCell = { type: 'text', cellContents: '' };
     switch (key) {
       default:
         return transactionItem[key];
       case 'totalQuantity':
         return {
-          type: this.props.transaction.isFinalised ? 'text' : 'editable',
-          cellContents: transactionItem.totalQuantity,
+          type: isFinalised ? 'text' : 'editable',
+          cellContents: totalQuantity,
         };
       case 'remove':
         return {
           type: 'checkable',
           icon: 'md-remove-circle',
-          isDisabled: this.props.transaction.isFinalised,
+          isDisabled: isFinalised,
         };
+      case 'doses': {
+        // If not a vaccine item, render an empty cell.
+        // If a vaccine, display the doses of the item. Only
+        // editable if doses > 1.
+        const { item } = transactionItem;
+        const { isVaccine, doses } = item;
+        const hasDosesOfOne = doses === 1;
+        if (!isVaccine) return emptyCell;
```

Comment:
if doses is 0 or null should we also make it non enterable and empty ?

Reply:
done a2ea237729812a9de505ac87f1d62ba68b8f8a6b - just going to treat a vaccine with doses nto set correctly as not a vaccine, and rendering an empty cell for doses?